demos/fleet_provisioning/fleet_provisioning_csr/demo_config.h

```diff
@@ -0,0 +1,250 @@
+ */
+#define AWS_MQTT_PORT    ( 8883 )
+
+/**
+ * @brief Path of the file containing the server's root CA certificate.
+ *
+ * This certificate is used to identify the AWS IoT server and is publicly
+ * available. Refer to the AWS documentation available in the link below
+ * https://docs.aws.amazon.com/iot/latest/developerguide/server-authentication.html#server-authentication-certs
+ *
+ * Amazon's root CA certificate is automatically downloaded to the certificates
+ * directory from @ref https://www.amazontrust.com/repository/AmazonRootCA1.pem
+ * using the CMake build system.
+ *
+ * @note This certificate should be PEM-encoded.
+ * @note This path is relative from the demo binary created. Update
+ * ROOT_CA_CERT_PATH to the absolute path if this demo is executed from elsewhere.
+ */
+#ifndef ROOT_CA_CERT_PATH
+    #define ROOT_CA_CERT_PATH    "certificates/AmazonRootCA1.crt"
+#endif
+
+/**
+ * @brief Path of the file containing the client certificate. This will be used
+ * as the provisioning certificate.
+ *
+ * Refer to the AWS documentation below for details regarding client
+ * authentication.
+ * https://docs.aws.amazon.com/iot/latest/developerguide/client-authentication.html
+ *
+ * @note This certificate should be PEM-encoded.
+ *
+ * #define CLIENT_CERT_PATH    "...insert here..."
+ */
+
+/**
+ * @brief Path of the file containing the client's private key.
+ *
+ * Refer to the AWS documentation below for details regarding client
+ * authentication.
+ * https://docs.aws.amazon.com/iot/latest/developerguide/client-authentication.html
+ *
+ * @note This private key should be PEM-encoded.
+ *
+ * #define CLIENT_PRIVATE_KEY_PATH    "...insert here..."
+ */
+
+/**
+ * @brief Path of the file containing the private key to provision.
+ *
+ * Refer to the AWS documentation below for details regarding client
+ * authentication.
+ * https://docs.aws.amazon.com/iot/latest/developerguide/client-authentication.html
+ *
+ * @note This private key should be PEM-encoded.
+ *
+ * #define PROVISIONING_KEY_PATH    "...insert here..."
+ */
+
+/**
+ * @brief Path of the file containing the CSR for the private key to provision.
+ *
+ * Refer to the AWS documentation below for details regarding client
+ * authentication.
+ * https://docs.aws.amazon.com/iot/latest/developerguide/client-authentication.html
+ *
+ * @note This CSR should be PEM-encoded.
+ *
+ * #define PROVISIONING_CSR_PATH    "...insert here..."
+ */
+
+/**
+ * @brief Path to which to write the client certificate obtained from the AWS
+ * IoT Fleet Provisioning service.
+ *
+ * Refer to the AWS documentation below for details regarding client
+ * authentication.
+ * https://docs.aws.amazon.com/iot/latest/developerguide/client-authentication.html
+ *
+ * @note This certificate should be PEM-encoded.
+ *
+ * #define PROVISIONING_CERT_PATH    "...insert here..."
+ */
+
+/**
+ * @brief Predefined template name.
+ *
+ * This is the name of the provisioning template to use.
```

Comment:
More information about the provisioning template setup will be helpful for customers.
```suggestion
 * @brief Name of the Fleet Provisioning template created in the AWS IoT account. 
 *
 * For more information on creating a provisioning template, refer to the following documentation:
 * https://docs.aws.amazon.com/iot/latest/developerguide/provision-template.html#fleet-provision-template
```